### ISSUE TITLE
There is no guidance on how to install Ionic native and Cordova plugin

### ISSUE THREAD

**Issue by Sampath-Lokuge** [ionic-team/ionic-docs#1233]
Hi,

There is no guidance on how to install Ionic native and Cordova plugin.

e.g. https://ionicframework.com/docs/native/sqlite

We need to have it under the installation section. 

e.g.

```
npm install @ionic-native/sqlite
 
ionic cordova plugin add cordova-sqlite-storage
```

**Comment by perrygovier** [comment]
Thanks! I'm looking in to this now. It looks like it's related to Stencil and slotted content.

**Comment by Sampath-Lokuge** [comment]
Great! It was there in the past. But now it is not?

**Comment by perrygovier** [comment]
Fixed in #1fb1f7e658e4cbcc4cdf92534b7bdde3f3967b51 Should be live in ~10-20 mins. 

The problem was the `slot="Community"` property served no purpose and was technically invalid markup. We used to get away with it, but in the newer version of Stencil, it refuses to show slotted content that has a name when the component uses default slots.

**Comment by Sampath-Lokuge** [comment]
Yes, now it shows for the Community but not for Enterprise?
